File: src/main/java/com/hank/guess/GameFor.java
package com.hank.guess;

import java.util.Random;
import java.util.Scanner;

public class GameFor {
    public static void main(String[] args) {
/*        int sum = 0;
        for (int i=1; i<=3; i++){
            sum += i; // sum = sum + i
        }
        System.out.println(sum);*/
        int secret = new Random().nextInt(10)+1;
        System.out.println("secret: " + secret);
        Scanner scanner = new Scanner(System.in);
        for (int i=1; i<=4; i++){
            System.out.print("Please enter a number(" + i + "/4):");
            int num = scanner.nextInt();
            System.out.println("第" + i + "次： " + num);
            if (num > secret) {
                System.out.println("Lower");
            } else if (num < secret) {
                System.out.println("Higher");
            } else {
                System.out.println("Greate,the number is: " + num);
                break;
            }
        }
        System.out.println("結束");
    }
}

File: src/main/kotlin/com/hank/GameFor.kt
package com.hank

import java.util.Random
import java.util.Scanner

fun main() {
    val secret = Random().nextInt(10)+1
    println("secret:$secret")
    val scanner = Scanner(System.`in`)
    var number = -1
    for (i in 1..5){
        print("Please enter a number(${i}/4): ")
        number = scanner.nextInt()
        println("第${i}次:$number")
        if (number > secret) {
            println("Lower")
        } else if (number < secret) {
            println("Higher")
        } else {
            println("Greate,the number is $secret")
            break
        }
    }
    if (number != secret ) println("可惜都沒猜中，secret是： $secret")
}
